src/mame/drivers/mpu4plasma.cpp

```diff
@@ -11,10 +11,12 @@
 #include "cpu/m68000/m68000.h"
 #include "screen.h"
 
-#include "mpu4plasma.lh"
 
+INPUT_PORTS_EXTERN( mpu4 );
```

Comment:
It’s really better to keep these declarations in the header associated with the source file where they’re defined, or they can get duplicated, out-of-sync, or stick around after the definition has been removed.